### DISCUSSION TITLE
StatusBarStyle not exported from  @capacitor/core@3.0.0-alpha.4 and @capacitor/core@3.0.0-alpha.5

### DISCUSSION THREAD

**Discussion by santoshshinde2012** [ionic-team/capacitor#3650] [General] [answered]
I tried to use `StatusBarStyle /FilesystemEncoding` from `@capacitor/core` but its giving me error like its undefined/not exported from `capacitor/core`.

Is there any specific solution are exist to resolve this one ?

**Comment by jcesarmobile** [comment, marked as answer]
For Capacitor 3 we are moving plugins out and putting them here https://github.com/ionic-team/capacitor-plugins.
Each one of them will be an independent package.

But those two are still unmerged
Status Bar https://github.com/ionic-team/capacitor-plugins/pull/58
Filesystem https://github.com/ionic-team/capacitor-plugins/pull/19
